CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to teamcity-rest-client library will be documented in this file.
 
+## [Unreleased]
+- Added ability to fetch builds' snapshot dependencies
```

Comment:
this is not entirely correct, it's build configurations' dependencies, not build dependencies. Build dependencies would be another builds